apps/src/lib/config/genesis.rs

```diff
@@ -233,6 +232,16 @@ pub mod genesis_config {
         // Hashes of whitelisted txs array. `None` value or an empty array
         // disables whitelisting.
         pub tx_whitelist: Option<Vec<String>>,
+        /// Expected number of epochs per year
+        pub epochs_per_year: u64,
+        /// PoS gain p
+        pub pos_gain_p: Decimal,
+        /// PoS gain d
+        pub pos_gain_d: Decimal,
+        /// PoS staked ratio
+        pub staked_ratio: Decimal,
+        /// PoS inflation rate last epoch
+        pub pos_inflation_amount: u64,
```

Comment:
I'm not sure if `staked_ratio` and `pos_inflation_amount` should be set in genesis config. I think `staked_ratio` should be calculated from the genesis total supply and validator stakes. `pos_inflation_amount` should probably be omitted on genesis as there is no last epoch

Reply:
Ok makes sense. FWIW, `staked_ratio` is still calculated from the genesis supply in `init_chain.rs` in branch `brent/pos-inflation-rewards` (#714)